eos-enable-zram

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+# Copyright (C) 2015 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+[ $# -eq 1 ] || exit -1
+[ "${1:-0}" -ge 0 ] || exit -1
```

Comment:
Can you add error messages for these so we find something in the journal if things get screwed up? E.g.,
```
if [ $# -lt 1 ]; then
  echo "error: no size supplied" >&2
  exit 1
fi
if [ "${1:-0}" -lt 0 ]; then
  echo "error: invalid size $1" >&2
  exit 1
fi